clients/cobol-lsp-vscode-extension/src/__tests__/JavaVersionCheckTest.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2019 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+import { DefaultJavaVersionCheck } from "../JavaVersionCheck";
+
+describe("Check Java version", () => {
+    it("Checks if Java version is supported", async () => {
+        var javaCheck = new DefaultJavaVersionCheck();
+        expect(javaCheck.isJavaVersionSupported('openjdk version "1.8.0-internal"')).toBeTruthy;
```

Comment:
toBeTruthy is function, you should call It to check the result. Use toBeTruthy();